packages/lib/modules/tokens/TokenInput/TokenInput.tsx

```diff
@@ -282,6 +282,7 @@ export const TokenInput = forwardRef(
                 shadow="none"
                 title={inputTitle}
                 type="number"
+                step="any" // is this okay? if not, user forced to integer input for lpb creation
```

Comment:
don't really understand  what you mean here. Were the user not able to add decimals before?

Reply:
yeah it was forcing this strange behavior and not letting me click next

![image](https://github.com/user-attachments/assets/76ad2ec4-4ccc-4dba-9a7d-cd009dc8edb9)

Reply:
wait you are able to use decimals and proceed to next after removing the `step="any"` from the input?